src/main/groovy/eu/plumbr/integration/artifactory/ArtifactoryClient.groovy

```diff
@@ -0,0 +1,6 @@
+package eu.plumbr.integration.artifactory
+
+interface ArtifactoryClient {
+
+  List<String> releasedIssues(String buildName, String buildNumber)
```

Comment:
I somehow don't like this overwhelming usage of `getX` and `setX`. A matter of taste, yes, but... I don't know...